elastic/kibana
# [Infra] Metadata: Fix the bug with nesting - ignore values nested > 3th level

## Summary

This is probably an edge case because it shouldn't happen if the metadata is in the defined structure. I saw this bug while testing with metricbeat (with oblt cluster it's not reproducible and some of the metrics were present in the metadata in this format: 
```
      info: {
        container: {
        .........
          network: {
            ingress: {
              bytes: 1410228770498, // ignore
            },
            egress: {
              bytes: 23527514469, // ignore
            },
          },
       ..........
        },
```
In theory, we should not have this data there based on the metadata type and as it's a nested property it returned an error.

https://github.com/elastic/kibana/assets/14139027/55b54ae4-63df-4574-bd93-ea33d3bc97f4

It happens because we have `container` > `network` > `ingress` > **bytes** and in metadata we don't support more than 3-level nesting - so the values can be like in this example: `container` > `image` > **name** - but the name should not be an object (it's a string in this case).

This can happen also for other asset types in case of some unsupported metadata format. To make sure that we skip those values in such cases I added a check to skip them when we prepare the metadata for the table.

## Testing
- Use metricbeat to monitor docker containers
- Open the new container view and select the metadata tab
   -  The table should be displayed without an error
   - There should not be an error even if the autorefresh is enabled in inventory
